model/src/main/java/org/cbioportal/model/GeneticProfile.java

```diff
@@ -20,7 +20,16 @@
         PHOSPHORYLATION,
         PROTEIN_LEVEL,
         PROTEIN_ARRAY_PROTEIN_LEVEL,
-        PROTEIN_ARRAY_PHOSPHORYLATION;
+        PROTEIN_ARRAY_PHOSPHORYLATION, 
+        GENESET_SCORE;
+    }
+    
+    public static class DataType {
```

Comment:
I found enum to be a bit annoying when it comes to values with `-` . These statics are used (I think in all cases) for guiding logic that is specific for one particular data type at a time. Anyway, do you have tips for how to deal with this if we would really want to use enum here?